content/_snippets/rippled-api-links.md

```diff
@@ -70,7 +70,7 @@
 [hexadecimal]: https://en.wikipedia.org/wiki/Hexadecimal
 [Interledger Protocol]: https://interledger.org/
 [RFC-1751]: https://tools.ietf.org/html/rfc1751
-[ripple-lib]: https://github.com/ripple/ripple-lib
+[ripple-lib]: https://github.com/XRPLF/xrpl.js
```

Comment:
I think we'll want to change the link text (in all the various places it gets used) when the renamed library is officially released. But this is fine for now.